doc/api/path.md

```diff
@@ -431,6 +431,39 @@ added: v0.11.15
 The `path.posix` property provides access to POSIX specific implementations
 of the `path` methods.
 
+*Note*: Be careful when using the following functions directly on Windows.
```

Comment:
This note should be moved into the bodies of each of the API sections. Sitting above it, it is far more likely that the Note will become disconnected...